doc/users/next_whats_new/2017-12-05_cividis_colormap.rst

```diff
@@ -0,0 +1,13 @@
+Cividis colormap
+----------------------------
+
+A new colormap, named cividis, has been optimized so that it looks similar to people with a red-green color vision deficiency. For more information, please see our paper (to be published in 2018).
+
+plot::
```

Comment:
I think this needs to be `. plot::`

Reply:
Just updated. Is the change I applied what you had meant?